Cargo.toml

```diff
@@ -13,6 +13,7 @@ members = [
     "core/primitives",
     "core/store",
     "runtime/runtime",
+    "runtime/near-runtime-fees",
```

Comment:
Does this need to have its own crate?

Reply:
So that we can refer to it from `near-vm-logic` without polluting it with receipt-specific info too much. Might move it inside later.